public/script.js

```diff
@@ -1,3 +1,36 @@
-const search=document.getElementById('search');
-const submit=document.getElementById('submit');
+const input = document.getElementById("search");
 
+const search = document.getElementById("submit");
+
+const req = (url, cb) => {
+  fetch(url)
+    .then(response => response.json())
+    .then(data => cb(data))
+    .catch(error => error);
+};
+
+req("/search", data => {
+  if (input.value === "") {
+    alert("please enter the country");
+  }
+});
+
+//
+// search.addEventListener("click", event => {
```

Comment:
Please delete these comments. Even if you're using them later, you shouldn't merge large comment blocks into master.

Reply:
There's a lot more than dom happening here and the PR name doesn't reflect the PR content.